### PR TITLE
Add mention all switch in URL parameter

### PR DESCRIPTION
**Pull request by relkli**
Hi guys,

This feature is for those who don't need to mention everyone in each alert, only high severity alert needs to take a look.

And I adjust the message template for print detail alert messages in the room.

### CODE REVIEW

**Comment by aledegano** [comment]
Thank you for your PR. I'll review it ASAP.

File: main.go [hunk 1]
```diff
@@ -144,6 +146,11 @@ func sendAlert(data alerttemplate.Alert, getParams url.Values) error {
 		return err
 	}
 
+	mention, err := url.Parse(getParams.Get("mention"))
```
Comment (aledegano) [inline main.go]: I'd call this variable `mentionAll` to give a better idea of what it's referring to.

**Comment by aledegano** [comment]
As is this PR would change the default behavior to **not** include all users.
Could you please invert the default `mention` to `true` so that by default all users are mentioned, and it can be turned off when desired?

File: main.go [hunk 2]
```diff
@@ -30,12 +30,14 @@ var logFormat = flag.String("log-format", "json", "can be empty string or json")
 var logLevel = flag.String("log-level", "info", "Can be one of:"+strings.Join(validLogLevels(), ","))
 var templateString = flag.String("template-string", messageTemplate, "template for the messages sent to hangouts chat")
 
-var messageTemplate = `<users/all>
-*{{.QueryParams.Get "env" | toUpper }}: {{ .Labels.alertname }} - {{.Status | toUpper}}*
+var messageTemplate = `*{{ .Labels.alertname }} - {{.Status | toUpper}}*
+{{ range .Labels.SortedPairs -}}
```
Comment (aledegano) [inline main.go:35]: Just for my interest: can you please explain me what's the benefit of this sort?

**Comment by relkli** [comment]
> As is this PR would change the default behavior to **not** include all users.
> Could you please invert the default `mention` to `true` so that by default all users are mentioned, and it can be turned off when desired?

Thanks for the review!
I'm inverted mentionAll behavior in latest commit, please review the changes, thanks!

File: main.go (hunk 2 above)
Comment (relkli) [reply main.go:35]: > Just for my interest: can you please explain me what's the benefit of this sort?

I think no additional benefit when it sorted, is there any good practice to iterate .Labels?

**Review by aledegano — APPROVED** [review]

**Review by odormond — APPROVED** [review]

**Pull request merged**